### ISSUE TITLE
Non printable characters aren't substituted.

### ISSUE THREAD

**Issue by Phlosioneer** [antirez/kilo#29]
In editorUpdateRow, a check is never performed to find non-printable characters. However, the function _should_ replace non-printable characters with '?'.

**Comment by practicalswift** [comment]
@Phlosioneer Good point! Addressed in commit https://github.com/kilofork/kilo/commit/2173454e3daed43c0be89f892ec0fd68276a7f78 in the `kilofork` project :-)
